router.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const express = require('express');
+const router = express.Router();
+const bcrypt = require('bcrypt');
+const basicAuth = require('./middleware/basic');
+const { Users } = require('./models/index');
+
+router.post('/signup', async (req, res, next) => {
+    try {
+        req.body.password = await bcrypt.hash(req.body.password, 5);
+        const record = await Users.create(req.body);
+        res.status(200).json(record);
+    } catch (error) {
+        res.status(403).send('Cannot Create User');
+    }
+});
+
+router.post('/signin', basicAuth, (req, res, next) => {
```

Comment:
You are calling the basicAuth middleware, but currently have nothing in the file.